playbooks/ansible-test-units-base/post.yaml

```diff
@@ -1,8 +1,12 @@
 ---
 - hosts: controller
   tasks:
+    # - name: Copy report file
+    # shell: "cp /home/zuul/.ansible/collections/ansible_collections/openvswitch/{{ _collection_namespace.stdout }}/tests/output/reports/coverage/index.html {{ ansible_user_dir }}/zuul-output/logs/controller/coverage/"
+
     # TODO: Migrate to fetch-zuul-logs when
     # https://review.openstack.org/#/c/583346/ is merged.
+
     - name: Collect log output
```

Comment:
For here, try setting https://docs.ansible.com/ansible/latest/modules/synchronize_module.html#parameter-copy_links

So we copy symlinks, not the symlinks themself